appinventor/codelabs/i-have-a-dream/index.html

```diff
@@ -0,0 +1,235 @@
+        <img src="img/if_else.png">
+        <p>Place the if-else block into the <code>when MLKButton.Click</code> event handler. Fill in the if-else block with the following blocks as shown.</p>
+        <img src="img/if_else_blocks.png">
+        <aside class="note">
+          <p>Conditional statements are fundamental to programming. They allow the program to make decisions based on the current state of the program. Almost all "if" statements check the values of component properties or values. In this case, the Player component's IsPlaying property is checked, and the program chooses an action based on whether the Player is playing or not.</p>
+        </aside>
+
+      </google-codelab-step>
+
+      <google-codelab-step label="Add code to pause the other speech">
+        <p>Now, we want to ensure that both speeches will never play at the same time. If the MLK speech is playing and we click the Malcolm X button, we want the MLK speech to pause.</p>
+        <p>To do this, all we need to do is add a <code>call MalcolmPlayer.Pause</code> block to the event handler as shown.</p>
+        <img src="img/pause_other.png">
+        <h2>Challenges</h2>
+        <ol>
+          <li>Code the Malcolm X button exactly like the MLK button. We want both buttons to behave the exact same way.</li>
+          <p><i>Hint: You can copy-paste blocks, as well as change the blocks to refer to different components using the arrow next to the component name.</i></p>
+        </ol>
+        <aside class="callout">
+          <h2>Test your app</h2>
+          <p>Try playing the two speeches. Can you pause a speech once it's playing? If a speech is playing, will clicking the other button pause that speech and start the other speech?</p>
+        </aside>
+
+      </google-codelab-step>
+
+      <google-codelab-step label="Checkoff #3">
+        <aside class="special">
+          <h2>i-have-a-dream-3</h2>
+          <p><b>To recieve this checkoff, you must</b></p>
+          <ul class-"checklist">
+            <li>Demonstrate your working app</li>
+            <li>Explain how you coded the Malcolm X button</li>
+            <li>Describe the similarities and differences between the code for the MLK button and the code for the Malcolm X button</li>
+            <li>Explain why we modified the <code>if</code> block to include an <code>else</code> block</li>
+            <li>Explain how the program knows a speech is playing</li>
+            <li>Name the different things that can happen when a Button is clicked</li>
+          </ul>
+        </aside>
+
+      </google-codelab-step>
+
+      <google-codelab-step label="Congratulations!">
+        <p>You have used App Inventor to create an app that can play two different speeches without overlap.</p>
```

Comment:
You have used App Inventor to create an app that can play two different speeches!